tasks/quality_control/task_quast.wdl

```diff
@@ -5,6 +5,7 @@ task quast {
     File assembly
     String samplename
     String docker = "quay.io/staphb/quast:5.0.2"
+    Int min_contig_len = 500
```

Comment:
will this interfere with the other workflows that use quast?

Reply:
It shouldn't as that's already the default value implemented when you don't set that option for quast's command.